docs/release/index.rst

```diff
@@ -9,24 +9,5 @@ Release Notes
 .. toctree::
    :maxdepth: 2
    :glob:
-   
-   v2.0_series
-   v1.11_series
-   v1.10_series
-   v1.9_series
-   v1.8_series
-   v1.7_series
-   v1.6_series
-   v1.5_series
-   v1.4_series
-   v1.3_series
-   v1.2_series
-   v1.1_series
-   v1.0_series
-   v0.9_series
-   v0.8_series
-   v0.7_series
-   v0.6_series
-   v0.5_series
-   v0.2_series
-   v0.1_series
+
+   ./v*
```

Comment:
Does this keep the order starting at the more recent versions? Otherwise you may want to take from my recent commit instead